client/src/app/uptime/monitors/components/dialog/AdvancedTab.tsx

```
@@ -0,0 +1,196 @@
+                <FormDescription>Override the default user agent string</FormDescription>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+
+      <FormField
+        control={form.control}
+        name="httpConfig.headers"
+        render={({ field }) => {
+          // Use local state to manage headers array
+          const [localHeaders, setLocalHeaders] = React.useState<Array<{ key: string; value: string; id: string }>>(() => {
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)